openquake/server/settings.py

```diff
@@ -89,7 +89,7 @@
 
 MIDDLEWARE_CLASSES = (
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
Please add a comment. i.e. `disabled because....`
or
is it possible to keep it on and disable (or decorate) only the affected views?